src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -171,6 +178,37 @@ UserRecord getUserByPhoneNumber(String phoneNumber) throws FirebaseAuthException
     return new UserRecord(response.getUsers().get(0), jsonFactory);
   }
 
+  /**
+   * @pre identifiers != null
```

Comment:
These are verified in the calling code. (As an alternative, we could just repeat the validation here.)

Reply:
Let's drop the `pre` annotations. That's not really a convention we use. It is understood that the arguments passed to this class have already been validated. If you want add a class-level javadoc to indicate that explicitly.